cs-property-service/src/main/java/org/egov/cpt/config/PropertyConfiguration.java

```diff
@@ -91,6 +91,9 @@ public MappingJackson2HttpMessageConverter jacksonConverter(ObjectMapper objectM
 	@Value("${persister.cancel.property.assessment.topic}")
 	private String cancelPropertyAssessmentTopic;
 
+	@Value("${persister.save.rent.demand}")
```

Comment:
Missing property file. This will application to not start up.

Reply:
This need not be used. We can just use property update topic.

Reply:
removed persister.save.rent.demand and using property update topic